### PR TITLE
Add issue, pull request, contributing templates

### PR DESCRIPTION
**Pull request by diemonster**
Addresses https://github.com/quintilesims/layer0/issues/260.

**Checklist**

- [x] docs completed

**What does this Pull Request Do?**

Adds templates for contributors, issues, and pull requests in the .github/ directory. 
Also adds CNAME file in /docs to maintain our project settings.

**Where should the reviewer start?**

See doc changes

**How should this be manually tested?**

N/A

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+### Actual behavior
+
+Exceptionally long log output should be posted as a gist (https://gist.github.com).
+
+### Steps to reproduce the behavior
+
+Please include the following if applicable:
+
+- [ ] Manual steps to reproduce issue
+- [ ] The output of `l0 admin debug`
```
Comment (zpatrick) [inline .github/ISSUE_TEMPLATE.md]: Sensitive material! How about we either take the `token` out of `l0 admin debug` or have this just be bullets for the api and cli versions

Comment (zpatrick) [inline .github/ISSUE_TEMPLATE.md]: [nit] Lets put layer0 version(s) on the top of the bug

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+Addresses # .
```
Comment (zpatrick) [inline .github/PULL_REQUEST_TEMPLATE.md]: :-1: 

```
**What does this pull request do?**
Include a short summary of what this PR intends to fix and/or a reference to an existing issue.


**How should this be tested?**
Include steps to test intended functionality.


** Checklist **
- [ ] Unit tests
- [ ] Smoke tests (if applicable) 
- [ ] System tests (if applicable) 
- [ ] Documentation (if applicable) 


closes #
(reference the issue(s) this pull request closes)
```

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (diemonster) [inline .github/ISSUE_TEMPLATE.md]: We mention in the CONTRIBUTING.md to remove sensitive material from issues, but I can see them still slipping in here I guess. I can remove this bit for now or add clarification not to put in tokens

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (diemonster) [inline .github/PULL_REQUEST_TEMPLATE.md]: 👎  with no explanation as to why will be added to the COMMENTS.md section 😉

**Comment by zpatrick** [comment]
:+1:

**Pull request merged**